src/main/java/com/inqoo/trainingservice/app/offer/OfferController.java

```diff
@@ -0,0 +1,29 @@
+package com.inqoo.trainingservice.app.offer;
+
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@CrossOrigin(allowedHeaders = "*", origins = "*")
+@RequestMapping("/offer")
+class OfferController {
+
+    private final OfferService offerService;
+
+    public OfferController(OfferService offerService) {
+        this.offerService = offerService;
+    }
+
+    @GetMapping
+    private List<Offer> getAllOffers() {
+        return offerService.getAll();
+    }
+
+    @PostMapping(value = "/{catId}/{subCatId}/{courseId}/{customerId}")
+    private Offer saveNew(@PathVariable("catId") Long catId, @PathVariable("subCatId") Long subCatId, @PathVariable(
```

Comment:
Consider to change passing parameters via some DTO that will represent input arguments (all required IDs).

Reply:
next time to fix